docs/client_reference.rst

```diff
@@ -1474,7 +1474,11 @@ Response object
 
          Returns value is ``'application/octet-stream'`` if no
          Content-Type header present in HTTP headers according to
-         :rfc:`2616`. To make sure Content-Type header is not present in
+         :rfc:`2616`. If *Content-Type* header is invalid (e.g., ``jpg``
+         instead of ``image/jpeg``), the value is ``text/plain`` by default
+         according to :rfc:`2045`.
```

Comment:
```suggestion
         :rfc:`2616`. If the *Content-Type* header is invalid (e.g., ``jpg``
         instead of ``image/jpeg``), the value is ``text/plain`` by default
         according to :rfc:`2045`. To see the original header check
         `resp.headers['CONTENT-TYPE']`.
```